plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1349,31 +1358,34 @@ public void testEmptyTimestampClusteringKey()
     public void testNestedCollectionType()
     {
         String keyspaceName = "keyspace_test_nested_collection" + randomNameSuffix();
-        createKeyspace(keyspaceName);
-        assertContainsEventually(() -> computeActual("SHOW SCHEMAS FROM cassandra"), resultBuilder(getSession(), createUnboundedVarcharType())
-                .row(keyspaceName)
-                .build(), new Duration(1, MINUTES));
-
-        session.execute("CREATE TABLE " + keyspaceName + ".table_set (column_5 bigint PRIMARY KEY, nested_collection frozen<set<set<bigint>>>)");
-        session.execute("CREATE TABLE " + keyspaceName + ".table_list (column_5 bigint PRIMARY KEY, nested_collection frozen<list<list<bigint>>>)");
-        session.execute("CREATE TABLE " + keyspaceName + ".table_map (column_5 bigint PRIMARY KEY, nested_collection frozen<map<int, map<bigint, bigint>>>)");
+        try {
```

Comment:
I think we don't need to create a new schema in this test. Can we create a table in the default namespace instead?

Reply:
ya instead we will have randomize the tables. I will address that as a followup. can we merge this PR?